### PR TITLE
Reconstructed Code & Object classes, added Scene class

### PR DESCRIPTION
**Pull request by frankhjwx**
removed 'easing' variable in objects, and we can add commands more humanlikely.
added scene class for group command changes. 
e.g.:
M(2,1000,2000,320,240,330,250) √  ->  M,2,1000,2000,320,240,330,250
M(1000,2000,320,240,330,250) √  -> M,0,1000,2000,320,240,330,250
M(1000,2000,320,240) √ ->  M,0,1000,2000,320,240
M(1000,320,240) √ -> M,0,1000,,320,240

added remove func from 0.0.2 branch to support remove commands.
e.g. obj.remove('M') will remove all M commands from obj.

### CODE REVIEW

File: Storyboard Engine/StoryboardObject.py [hunk 1]
```diff
@@ -6,12 +6,12 @@
 
 # Old Class, will be modified later
 class Object:
-    def __init__(self, file_name, type='Sprite', origin='Centre', x=320, y=240,
+    def __init__(self, file_name, layer='Foreground', type='Sprite', origin='Centre', x=320, y=240,
```
Comment (Mahua233) [inline Storyboard Engine/StoryboardObject.py]: exchange layer and type? 'Sprite,"layer","origin","filepath",x,y' <- look at is

File: Storyboard Engine/StoryboardObject.py [hunk 2]
```diff
@@ -42,107 +42,184 @@ def add(self, x):
...
+            raise RuntimeError('Arg Num of Trigger Command is wrong.')
 
     def LoopOut(self):
-        self.currentLoopLevel -= 1
+        if self.currentLoopLevel > 0:
+            self.currentLoopLevel -= 1
+        else:
+            raise RuntimeError('Not in any loop!')
+
+    def Remove(self, key):
```
Comment (Mahua233) [inline Storyboard Engine/StoryboardObject.py]: Add a Remove by index func?

**Review by Mahua233 — COMMENTED** [review]
http://zh-google-styleguide.readthedocs.io/en/latest/google-python-styleguide/python_style_rules/#id16 好像考试大师你说要用规范来着 这边写着推荐Method Names lower_with_under()(Public) _lower_with_under() (protected) __lower_with_under() (private)

**Comment by frankhjwx** [comment]
等oriental blossom搞完就修

**Review by Mahua233 — APPROVED** [review]
ok

**Pull request merged**